File: Programming Basics/C++/Conditional Statements/WorldSwimmingRecord/WorldSwimmingRecord.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
    // 1. Read input
    double existingRecordInSeconds, distanceInMeters, timePerMeter;
    cin >> existingRecordInSeconds >> distanceInMeters >> timePerMeter;

    // 2. Calculate travel time
    double totalTimeInSeconds = distanceInMeters * timePerMeter + floor(distanceInMeters / 15) * 12.5;

    // 3. Print output
    cout.setf(ios::fixed);
    cout.precision(2);
    if (totalTimeInSeconds < existingRecordInSeconds)
        cout << "Yes, he succeeded! The new world record is " << totalTimeInSeconds << " seconds." << endl;
    else
        cout << "No, he failed! He was " << totalTimeInSeconds - existingRecordInSeconds << " seconds slower." << endl;
}

File: Programming Basics/JavaScript/Conditional Statements/Exercises/world-swimming-record.js
function solve(input) {
    const existingRecordInSeconds = Number(input[0]);
    const distanceInMeters = Number(input[1]);
    const timePerMeter = Number(input[2]);

    const totalTimeInSeconds = distanceInMeters * timePerMeter + Math.floor(distanceInMeters / 15) * 12.5;
    if (totalTimeInSeconds < existingRecordInSeconds) console.log(`Yes, he succeeded! The new world record is ${totalTimeInSeconds.toFixed(2)} seconds.`);
    else console.log(`No, he failed! He was ${(totalTimeInSeconds - existingRecordInSeconds).toFixed(2)} seconds slower.`);
}